Client/Helpers/FxALoginHelper.swift

```diff
@@ -55,6 +55,11 @@ class FxALoginHelper {
             return loginDidFail()
         }
 
+        // If we've got here then we're loading an already logged in account.
+        // At this point, no one is interested if the account is verified or not,
+        // so assume it is.
+        accountVerified = true
```

Comment:
So I have multiple thoughts on this:

1. We are only concerned about the app starting up _after_ it has been logged in and the stopped. You are right that the assumption that the account has been verified does not hold.
2. `FxALoginFlags` are short lived, and are used to tell the `FxALoginHelperDelegate` after a successful login.
3. At the point we're interested in, at `AppDelegate.application(_:didFinishLaunching:)`, there is no delegate.
4. Conceptually, if we do have a delegate at that point, then it would be nice to be able to rely on `FxALoginFlags` to be accurately reporting back the state of the account.
5. There is a `.needsVerification` enum value, though it's _possible_ that the account could've had its password changed since login and needs verification. 

In the end, I'm opting for: 
```swift
accountVerified = account.actionNeeded != .needsVerification
```

because though it doesn't cover all overlapping states, it does cover the `.needsVerification` state better that `.actionNeeded == .none`.